app/models/sample.rb

```diff
@@ -5,6 +5,7 @@ class Sample < ApplicationRecord
   include MetadataSortable
   include HasPuid
   include History
+  include HasPuid
```

Comment:
`HasPuid` is here twice

Reply:
Removed in 04c398de4f22fd148c83b83dfe08ff76fa02bd13. Looks like I missed it during rebasing